src/common/metrics/hospitalizations.tsx

```diff
@@ -129,7 +132,7 @@ function renderStatus(projections: Projections): React.ReactElement {
     'This suggests there is likely enough capacity to absorb a wave of new COVID infections',
     'This suggests some ability to absorb an increase in COVID cases',
     'This suggests hospitals may not be well positioned to absorb a wave of new COVID infections without substantial surge capacity. Caution is warranted',
-    'This suggests hospitals cannot absorb a wave of new COVID infections without substantial surge capacity. Aggressive action urgently needed',
+    'This suggests hospitals cannot absorb a wave of new COVID infections without substantial surge capacity.',
```

Comment:
I tweaked this copy since there are still a lot of counties in this bucket and I think the copy may be overly sensational.

cc/ @BrettBoval

Reply:
SGTM